contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1331,6 +1331,8 @@ fieldName: |-
   And it can be on multiple lines.
 ----
 - When truncating YAML, always precede an ellipsis with a pound so that the YAML is valid.
```

Comment:
I like that you included the demonstration of the format in the lines you added. Might as well add this clarification too :slightly_smiling_face: 
```suggestion
- When truncating YAML, always precede an ellipsis with a pound (`#`) so that the YAML is valid.
```

Reply:
Thanks, @jeana-redhat . Updated.